### ISSUE TITLE
Definition of floating base generalized acceleration

### ISSUE THREAD

**Issue by FenglongSong** [stack-of-tasks/pinocchio#1656]
Hi,

Thanks for the replies in https://github.com/stack-of-tasks/pinocchio/issues/1357 and https://github.com/stack-of-tasks/pinocchio/issues/1137, I got to know the generalized velocities are defined in local joint frames, which means for floating-base dynamics, the linear and angular velocities of the base are expressed in their own frames rather than world frame. Therefore, I'd like to confirm that is the generalized acceleration also expressed in the local frame of the joint?

That means, if I use `a = aba(model, data, q, v, tau)` to compute the generalized acceleration, `a` should be defined as:
`a = [linear_acceleration_in_body_frame, angular_acceleration_in_body_frame, joint_accelerations]`
Is that correct? I will appreciate your clarification! Thanks a lot.

**Comment by jcarpent** [comment]
Yes, you are correct. For the next question, I guess it would be better to use the discussion feature of GitHub to exchange on the topic. I think this might be of interest to others.
